cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -77,7 +77,7 @@ class PredicateFunctionsTest extends DocumentingTest {
     }
     section("any()", "functions-any") {
       p("""The function `any()` returns `true` if the predicate holds for at least one element in the given list.
-          #`null` is returned if the list is `null` or all of its elements are `null`.""".stripMargin('#'))
+          #`null` is returned if the list is `null` or if one of the comparisons evaluates to `null` and no other has evaluated to true.""".stripMargin('#'))
```

Comment:
My experiments show

```
1. RETURN any(x IN [null] WHERE x = 1) => null
2. RETURN any(x IN [null, 1] WHERE x = 1) => true
3. RETURN any(x IN [1, null] WHERE x = 1) => true
4. RETURN any(x IN [2,null] WHERE x = 1) => null
5. RETURN any(x IN [null, 2] WHERE x = 1) => null
6. RETURN any(x IN [1, null, 2] WHERE x = 1) => true
```

I think that matches with your sentence, so here I just have a suggestion of a better wording:

```suggestion
          #`null` is returned if the list is `null`, or if the predicate evaluates to `null` for at least one element and does not evaluate to `true` for any other element.""".stripMargin('#'))
```